src/librbd/io/ImageRequestWQ.h

```diff
@@ -72,10 +72,20 @@ class ImageRequestWQ
   void set_require_lock(Direction direction, bool enabled);
 
   void apply_qos_iops_limit(uint64_t limit);
+  void apply_qos_bps_limit(uint64_t limit);
```

Comment:
Nit: instead of 6 helper methods, perhaps just one that passes the limit and the ```RBD_QOS_XYZ``` flag.